stubs/networkx/networkx/algorithms/link_analysis/pagerank_alg.pyi

```diff
@@ -10,19 +11,19 @@ __all__ = ["pagerank", "google_matrix"]
 def pagerank(
     G: Graph[_Node],
     alpha: float | None = 0.85,
-    personalization: SupportsGetItem[Incomplete, Incomplete] | None = None,
+    personalization: Mapping[_Node, float] | None = None,
```

Comment:
Disclaimer: This and similar changes in this PR are are because the functions use more than `__getitem__` of the passed arguments and the parameters are documented as dictionaries. Covariant `Mapping` is a middle ground between a dict and a protocol of only used methods (relying on implementation details).